config/helpers.exs

```diff
@@ -27,16 +27,20 @@ defmodule Cog.Config.Helpers do
   end
 
   # Returns a list containing the enabled chat provider and all other providers.
-  def provider_list,
-    do: [Enum.find(@chat_providers, &enabled_provider?/1)] ++ @other_providers
+  # Or nil if no provider is specified.
+  def provider_list do
+    case Enum.find(@chat_providers, &enabled_provider?/1) do
+      nil -> nil
```

Comment:
Would it make more sense for this to be `[]` or maybe `@other_providers` instead of `nil`?